src/index.js

```diff
@@ -1,28 +1,28 @@
 /* eslint-disable */
```

Comment:
- [ ] [REQUIRED] Kindly ensure to enable Eslint on all `.js` files because Eslint helps to enforce best practices for coding, such as using const instead of var, avoiding unused variables, and checking for potential errors. This can help you write better, more efficient code that is easier to maintain and debug.

- [  ] Also, ESLint can enforce best practices for coding, such as using const instead of var, avoiding unused variables, and checking for potential errors. This can help you write better, more efficient code that is easier to maintain and debug.

- [ ] Fix all the Eslint errors manually, please.